### ISSUE TITLE
How to add dropout to a pretrained network?

### ISSUE THREAD

**Issue by kbenahmed89** [vlfeat/matconvnet#587]
Hello,

Im tuning a pretrained network CNN-F, I need to use dropout while training, what I did is inserting two layers after fc6 and fc7 this way:
net.layers{end+1} = struct('type', 'dropout', ...
                             'name', sprintf('dropout6'), ...
                             'rate', 0.5) ;
I set mode parameter in cnn_train to 'normal' while training and to 'test' while testing.

is this the correct way to do it?
